utils.py

```diff
@@ -1,4 +1,4 @@
 def get_personal_message() -> str:
```

Comment:
You're never copying `utils.py` into your Docker container. How will any of this code be accessible in the container, then?

Since you've shifted `main.py` into an `/app` folder, you should move `utils.py` to that folder as well, and update your imports accordingly.